### ISSUE TITLE
soreag:GeologicFeature

### ISSUE THREAD

**Issue by nicholascar** [ESIPFed/sweet#181]
## Preferred term label
geologic feature - current
Geologic Feature - proposed. Capitals for a class name

## Synonyms
Geological Feature - `skos:altLabel`. "Geological" is used in place of Geologic in some places such as Australia

## Textual definition
A Geologic Feature is a geospatial feature on the Earth that results from geological processes. 

## Usage Note
`skos:scopeNote`: Geologic Features include Stratigraphic Units, Non-stratigraphic (Lithodemic) units, Stratigraphic Event Features, Tectonic and Structural Features, GeoResource Accumulations, and Geologically Significant Sites.

## Suggested parent term
`sorea:PlanetaryRealm` - current
*restriction*: `sorel:hasRealm` **only** `sorea:Geosphere`

## Attribution
Using schema.org as well as FOAF for search engine recognition.
```
dcterms:contributor [
    a foaf:Organization , <http://schema.org/Organization> ;
    foaf:name "Geological Survey of Queensland" ;
    <http://schema.org/name> "Geological Survey of Queensland" ;
    <http://schema.org/identifier> <http://linked.data.gov.au/org/gsq> .
] ;
```

## example RDF (turtle)
```
@prefis sdo: <https://schema.org/> .

soreag:GeologicFeature 
    a owl:Class ;
    rdfs:subClassOf sorea:PlanetaryRealm ,
    [ 
        a owl:Restriction ;
        owl:onProperty sorel:hasRealm ;
        owl:allValuesFrom sorea:Geosphere
    ] ;
    rdfs:label "Geologic Feature"@en ;
    skos:definition "A Geologic Feature is a geospatial feature on the Earth that results from geological processes."@en ;
    skos:scopeNote "Geologic Features include Stratigraphic Units, Non-stratigraphic (Lithodemic) units, Stratigraphic Event Features, Tectonic and Structural Features, GeoResource Accumulations, and Geologically Significant Sites"@en ;
    dcterms:contributor [
        a foaf:Organization , sdo:Organization ;
        foaf:name "Geological Survey of Queensland" ;
        sdo:name "Geological Survey of Queensland" ;
        sdo:identifier <http://linked.data.gov.au/org/gsq> .
    ] ;
```

**Comment by nicholascar** [comment]
We  - Geological Survey of Queensland - are planning to define a series of subclasses for this Class if this definition is accepted. Some of the subclasses are mentioned in the scopeNote already: Craton, Orogen, Shield etc.

**Comment by dr-shorthair** [comment]
@smrgeoinfo you may like to check the defs here

**Comment by lewismc** [comment]
A pull request would be really helpful @nicholascar :)
If not, let me know and I can whip one up.

**Comment by smrgeoinfo** [comment]
your definition is broadly consistent with GeoSciML v4.0.1, so OK there, especially given the hodge-podge of things that are sub classes of 'geologic feature' in soreag  : 

> The abstract GeologicFeature class represents a conceptual feature that is hypothesized to exist coherently in the world.
>    * this corresponds with a "legend item" from a traditional geologic map
>    * while the bounding coordinates of a Geologic Feature may be described, its shape is not.

**Comment by smrgeoinfo** [comment]
some other observations:
all of the other classes have labels in lower case, so it would be odd if the label were 'Geologic Feature'.
Alt label suggestion is fine.

I note that 'geologic structure' is a subclass of 'geosphere',  I would argue that 'geologic structure' should be a subclass of 'geologic feature', consistent with GeoSciML usage and your text above.
It seems to me that most of the subclasses of 'geologic feature' are defined in tectonic terms, so labeling the class 'tectonic feature' seems more appropriate to me.  The conflation of partOf and kind of relations in the hierarchy is a problem, as well as duplication of concepts ('volcanic arc' vs. 'arc').

What's the benefit of defining more subclasses under 'geologic feature'?

**Comment by smrgeoinfo** [comment]
There is no definition of 'realm', but I'd guess the intention is something like [definition 3 in Merriam Webster](https://www.merriam-webster.com/dictionary/realm): 'a primary marine or terrestrial biogeographic division of the earth's surface', generalized to apply to any planet, in which case the I suppose subclassing 'geologic feature' like caldera, diapir, craton makes sense. 

Changing the parent class from PlantaryRealm  to 'GeometricObject' is a pretty big modification of the wordnet. Its a very 'GIS' kind of view--that the geometry is primary, rather than a feature oriented view in which location/shape are properties of the feature.  

In short on that point, I don't thing changing the parent class makes sense.

**Comment by dr-shorthair** [comment]
I would also caution against seeing Geologic Feature as primarily a kind of GeometricalObject. That is not how classical (historical) geologists think - the geometry is really a secondary property, compared to the primary _genetic_ conceptualization.

**Comment by nicholascar** [comment]
> @lewismc A pull request would be really helpful @nicholascar :)

I will put in a PR once the issues raised here are addressed

> @smrgeoinfo all of the other classes have labels in lower case, so it would be odd if the label were 'Geologic Feature'.

OK, I will rescind that proposal and stick to lower case. I'll add an upper case altLabel too

> @smrgeoinfo I note that 'geologic structure' is a subclass of 'geosphere', I would argue that 'geologic structure' should be a subclass of 'geologic feature', 

I agree but this is out of scope for this change request. When this is done, I propose to work on that and a range of others subclasses of Geologic Feature

> @smrgeoinfo It seems to me that most of the subclasses of 'geologic feature' are defined in tectonic terms, so labeling the class 'tectonic feature' seems more appropriate to me.

Yes, I think you are correct. I'm working with our (Queensland's) geologists to make a whole tectonic and other class hierarchy but I want to insert it into SWEET here at Geologic Feature. 

I think the way forward here is to generalise the text of Geologic Feature - make it not just about tectonics - and then to reveal the considered class hierarchy we've been working on.

> @smrgeoinfo What's the benefit of defining more subclasses under 'geologic feature'?

Subclasses are actually used by the geologists?

> @smrgeoinfo & @dr-shorthair "Changing the parent class from PlantaryRealm to 'GeometricObject' is a pretty big modification of the wordnet. Its a very 'GIS' kind of view..." & "I would also caution against seeing Geologic Feature as primarily a kind of GeometricalObject"

OK, I hear you both and will remove the link. I'm most concerned about having a definition for Geometric Object so propose to adopt GeoSciML's as per @smrgeoinfo's suggestion above.

> In short on that point, I don't thing changing the parent class makes sense.

As above, I'll drop the geometry association but I still don't like the `subClassOf` relation to Realm. Sure, `GeologicFeature`s are *within* a realm but are not sub classes of it. The `hasRealm` property does, I think cater for the association without the need for a subclassing.

Some of the other Realm subclassing also seems to suffer from the same issue: the `CriticalZone` is *within* the `PlanetaryRealm`, not a subclass of it.

So I support the small hierarchy of:

* Realm
  * PlanetaryRealm
  * StellarRealm

But not the children of PlanetaryRealm, StellarRealm or that OuterSpace should be a child of Realm. All `isPartOf` in some way, not `subClassOf`.


### Amended proposal
The propsal now really only adds a definition, mostly from GeoSciML.

```
@prefix sdo: <https://schema.org/> .

soreag:GeologicFeature 
    a owl:Class ;
    [ 
        a owl:Restriction ;
        owl:onProperty sorel:hasRealm ;
        owl:allValuesFrom sorea:Geosphere
    ] ;
    rdfs:label "geologic feature"@en ;
    skos:altLabel "Geologic Feature"@en ;
    skos:altLabel "Geological Feature"@en ;
    skos:definition "A geologic feature is a conceptual feature a that is hypothesized to exist coherently in the Earth that results from geological processes"@en ;
    skos:scopeNote "Geologic Features include Sedimentary Basins, Stratigraphic Units, Non-stratigraphic (Lithodemic) units, Stratigraphic Event Features, Tectonic and Structural Features, GeoResource Accumulations, and Geologically Significant Sites"@en ;
    dcterms:contributor [
        a foaf:Organization , sdo:Organization ;
        foaf:name "Geological Survey of Queensland" ;
        sdo:name "Geological Survey of Queensland" ;
        sdo:identifier <http://linked.data.gov.au/org/gsq> .
    ] ;
```

**Comment by dr-shorthair** [comment]
Regarding the arguments around sub-classing vs partition: it depends how the concept is to be used - individual vs class. The realms are mostly individuals (there is one terrestrial realm, one oceanic realm, one interstellar realm, etc), whereas feature/structure/geometricObject are clearly classes as they have many members. Different meta-levels.

**Comment by dr-shorthair** [comment]
So I agree that subsumption is almost certainly _not_ the right relationship between realms and structures - i.e. this hierarchy is wrong. 
![image](https://user-images.githubusercontent.com/608303/72770842-30637d00-3c53-11ea-9c1f-148f8b971f22.png)

**Comment by dr-shorthair** [comment]
I suspect that the underlying cause is that subsumption was almost the only form of relationship between classes within SWEET from the very beginning. It allows views like the one above to be presented, but within which the semantics of the nesting slides rather misleadingly between sub-classing, instantiation, and partonymy.

**Comment by nicholascar** [comment]
Since the discussions above seems not to have issue with the annotation properties, I've created PR #182.

**Comment by lewismc** [comment]
I updated https://github.com/ESIPFed/sweet/wiki/SWEET-Annotation-Convention... any comments?

**Comment by rduerr** [comment]
On the subject of multiple definitions, I am totally opposed to this!  Entities with different definitions need different URI's and axioms!  We need ontological mechanisms to say that thermokarst means blah, blah, blah to the permafrost community; while thermokarst means something entirely different to the geology community.  In ENVO we did this by labeling the terms differently - thermokarst process vs thermokarst landform.  However, it isn't at all clear that this will work in all cases.  It would be convenient to be able to call thermokarst a synonym for both versions.

**Comment by lewismc** [comment]
Hi @rduerr can you please post your comment at https://github.com/ESIPFed/sweet/issues/183 where we can sort this out? Thank you. Have a great weekend :)
